system/lib/libc/emscripten_time.c

```diff
@@ -25,6 +25,13 @@ void _localtime_js(const time_t *restrict t, struct tm *restrict tm);
 void _gmtime_js(const time_t *restrict t, struct tm *restrict tm);
 double _emscripten_date_now();
 double emscripten_get_now_res();
+void emscripten_get_timezone_js(char* buffer, int length);
```

Comment:
Can we add leading underscore to this function name so folks will know its internal.